### DISCUSSION TITLE
Account restricted for violation of terms of service

### DISCUSSION THREAD

**Discussion by AlexissCA** [codewars/codewars.com#3056] [General]
Hey, I got this on 2 of my accounts (of total 4, second blocked is sillyinsky), why? how to unrestrict the account? Restricted accounts: https://www.codewars.com/users/AlexissCode/stats, https://www.codewars.com/users/sillyinsky ; just noticed third one is also restricted: https://www.codewars.com/users/andywarhol/stats; please help! 

// update
all 4 accounts got restricted, please help!

**Comment by Blind4Basics** [comment]
and for what reason exactly do you need _four_ accounts in the first place...?

**Reply by AlexissCA** [reply]
hey!, thank you for response, some perfecionsm issues I think, as on codewars we can't have like practicing modes at one account:
first account was to start solving problems, this one: https://www.codewars.com/users/learnin so there is lots of really embarassing code
second one: https://www.codewars.com/users/AlexissCode, wanted to have some cleaner solutions, and this is like my main account,
third: https://www.codewars.com/users/sillyinsky - I created that as I had some break from solving katas and started from 0....
fourth: https://www.codewars.com/users/andywarhol/stats - this was created with the previous one and it has "cleanest" possible solution of the above, like adding missing semicolons;
So this is are like psychiatric issues more, than violating terms of services...

**Reply by Blind4Basics** [reply]
except that andywarhol has the _exact same_ solutions than sillyinsky, and  AlexisCode  has the _exact_ same solutions than learnin. With some crossing over ~~here and there~~ most of the time. There are no improvements there. At all.

**Reply by AlexissCA** [reply]
from my point of view, there is improvement, learnin account I mostly don't touch, sillyinsky is like my playground lately, I have even 5 solutions for problems, checking the efficiency on jsbench for each, trying to simplify it and come to cleanest solution which goes on AlexissCode and andywarhol than; please, this platform is important to me, it's like my almost daily workout routine

**Reply by Blind4Basics** [reply]
>  and come to cleanest solution which goes on AlexissCode and andywarhol

that's a lie, but anyway.

Delete learnin, andy and alexis accounts, rename the silly one whatever you want (or don't change it, it's up to you), and you have a deal (eg, "silly" gets unlocked)

**Reply by AlexissCA** [reply]
why can't you unlock my AlexissCode account - where I did most of katas? I will delete all other accounts. What is a lie? I copy my own solutions between my accounts...! I will delete learnin, sillyinsky, andywarhol, will you unrestrict AlexissCode than? I am soon to go on those nutts interviews, would really love to have well earned pretty codewars badge :(

**Reply by Blind4Basics** [reply]
* alexiscode has 600 extra solutions coming from nowhere, so no, you won't get this account back
* lie/not a lie. Whatever, forget about that.
* the account with which you actually earned a nice badge is the silly one, so you should keep this one. The others (out of learnin) are just fakes/false mirors.

do we have a deal? (if not, all 4 remain in their current state)

**Reply by AlexissCA** [reply]
what do you mean from nowhere??? so where do they come from? this is my most active account, that I work longest on - check dates. I wrote in first response sillyinsky and andywarhol I created after the break from solving katas therefore on AlexissCode I have more solved katas, as I just didn't freakin copy those 600 solutions from AlexissCode there, please dude, I loved the codewars and improving my skills there, now I feel like some criminal

**Reply by AlexissCA** [reply]
Please, I will remove all other 3 accounts, please, don't make this codewars experience horrible now

**Reply by Blind4Basics** [reply]
Lemme put that under a different light:
If you keep one of andy or alexis, _any_ mod crossing path with it will instantly lock it again and invalidate everything, because of what they look like (eg. copycat/cheater). So letting you go on with one of these is not a good thing for anyone.
Also, you should really consider using one single account. You make mistake/write bad code? congrats, you're human. Everyone of us do so. 

So, silly account it is (or learnin, but you have less solutions there, so I doubt you'd wanna keep that one), which, once again, you can then rename how you prefer.

**Reply by AlexissCA** [reply]
This is full lack of respect to someone's time, because you decided "it's a lie" - still don't get how it's a lie - 600 katas are from nowhere? - yeah, solved them, so what's the lie? Bring some reasonable argument.

**Reply by AlexissCA** [reply]
I want to stay with AlexissCode, and will change the name there, will delete all other accounts immediately

**Reply by AlexissCA** [reply]
codewars don't have possibility of having different modes of learning (as scary leetcode does) that's why I created 4 accounts but this is my business I was exchanging my own solutions between them, but fine, will stay with 1 account if that causes some mistrust and discussion like above

**Reply by Kacarott** [reply]
The reason is not due to any "lie". As Blind4Basics stated, unfortunately your account is absolutely full of flags that make it look like a cheaters account. There isn't any good way to undo that, If you stayed on the Alexis account, it could be that every week or so a mod would go through, locking down your account (which also takes considerable time), only for you to have to raise another issue, only for it to be re-opened (takes yet more time) on repeat.

I get that it sucks, but there isn't a good solution. Keeping the one account which is still "clean" is the best solution for everyone, it means that after some time, you can earn back the points you lost by solving more kata, and without any more problems. Staying on the other account will mean problems and time spent for everyone, no matter how long it goes.

**Reply by AlexissCA** [reply]
so yeah, solution is like on the account with less solved katas I will be recreating the progress I had on other account..., this is nutts..., but fine...

**Reply by AlexissCA** [reply]
is it somehow possible to copy all those solutions, just to even freakin see how I solved that before (doing progress or not)? - before deleting the account

**Reply by Kacarott** [reply]
Yeah, to be honest this is the first time I have encountered this situation too, and I agree its pretty sucky.

But I also notice that youve managed to solve over *1600* kata in the last year alone. I think it will only be a couple months before you've caught up and gone past the other account, and then will be glad that your account isnt constantly getting flagged for cheating 😅. But thanks for understanding.

**Reply by Blind4Basics** [reply]
automatically, you mean? no, sorry.
You can gather the solutions manually before deleting the account if you want, but I'd strongly suggest you don't use them directly in the remaining account otherwise you'd end up with 600 new flags on your clean account... xo

**Reply by AlexissCA** [reply]
not really understanding, just no choice:(, thanks for at least not call me cheater anymore...

**Reply by Blind4Basics** [reply]
oh, btw, the "silly account in the sky" is unlocked ('couldn't resist, sorry...)

**Reply by Kacarott** [reply]
The easiest way to reference your old solutions is to go here https://www.codewars.com/users/AlexissCode/completed_solutions

**Reply by AlexissCA** [reply]
ok, copying it manually is really dumb idea..., ok, deleting accounts :( RIP

**Comment by AlexissCA** [comment]
> "silly account in the sky"



> oh, btw, the "silly account in the sky" is unlocked ('couldn't resist, sorry...)
what you couldn't resist?

**Reply by Blind4Basics** [reply]
to make that pun, I meant. Yeah, that probably was a stupid idea of mine. Sorry...

**Reply by AlexissCA** [reply]
just hope it wasn't other words of disrespect from you; accounts deleted, silly name updated, how to avoid situations like that...? not to have the account degraded?

**Reply by Blind4Basics** [reply]
no, not at all.

To avoid this situation: just don't use several accounts, or keep them strictly unconnected (like, python with one, haskell with another, for example)

**Reply by AlexissCA** [reply]
ok..., we can close this crazy discussion

**Comment by Blind4Basics** [comment]
note: the new name, isn't really better, to say the least... Anyway...

**Reply by AlexissCA** [reply]
:), ok, so should be totally not connected, ok....

**Reply by Blind4Basics** [reply]
yeah... But again, better to _not_ have several accounts, right...?

**Reply by AlexissCA** [reply]
would have one from the start if I could have like one mode where I have solution I find "cleanest" or best other way, not to see previous mess I wrote...

**Reply by AlexissCA** [reply]
changed the name completely, hope this one is not already red flagged or smth

**Comment by Kacarott** [comment]
I think this is resolved.
